.evergreen/config.yml

```
@@ -363,6 +363,39 @@ functions:
           CRYPT_SHARED_LIB_PATH="${CRYPT_SHARED_LIB_PATH}" \
             sh ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
 
+  "run docker test":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          set -ex
+          cd $DRIVERS_TOOLS/.evergreen/docker
+          ENTRYPOINT=/root/test-entrypoint.sh bash run-local.sh
+          # Generate a test results file
+          cd ${PROJECT_DIRECTORY}
+          make test
+
+  "run oidc test":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${aws_test_secrets_role}
+    - command: shell.exec
+      type: test
+      params:
+        include_expansions_in_env: ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"]
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          set -ex
+          cd $DRIVERS_TOOLS/.evergreen/auth_oidc
+          export ENTRYPOINT=/root/docker_entry_base.sh
+          bash start_local_server.sh
```

```suggestion
          ENTRYPOINT=/root/docker_entry_base.sh bash start_local_server.sh
```
For consistency with the `run docker test` script.

Alternatively, consider using [EVG's `env` command parameter](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands/#shellexec) to move the env-var definition out of the script, e.g.:
```yaml
command: shell.exec
type: test
params:
  include_expansions_in_env: [...]
  env:
    ENTRYPOINT: /root/docker_entry_base.sh
  working_dir: "src"
  script: |
    ${PREPARE_SHELL}
    ...
```